.gitlab-ci.yml

```diff
@@ -37,26 +37,11 @@ variables:
 .docker-cache-status:              &docker-cache-status
   variables:
     CARGO_HOME:                    "/ci-cache/parity-ethereum/cargo/${CI_JOB_NAME}"
+    SCCACHE_REDIS:                 "redis://:${REDIS}@172.17.0.1/0"
   dependencies:                    []
   before_script:
     - rustup show
     - cargo --version
-    - SCCACHE_ERROR_LOG=/builds/parity/parity-ethereum/sccache_debug.log
-        RUST_LOG=sccache=debug
-        sccache --start-server
-    - sccache -s
-  after_script:
-    # sccache debug info 
-    - if test -e sccache_debug.log;
-      then
-        echo "_____All crate-types:_____";
-        grep 'parse_arguments.*--crate-type' sccache_debug.log | sed -re 's/.*"--crate-type", "([^"]+)".*/\1/' | sort | uniq -c;
-        echo "_____Non-cacheable reasons:_____";
-        grep CannotCache sccache_debug.log | sed -re 's/.*CannotCache\((.+)\).*/\1/' | sort | uniq -c;
-      else
-        echo "_____No logs from sccache_____";
-        exit 0;
-      fi
```

Comment:
so debugging will be disabled completely?

Reply:
Yep, sccache debugging was actually helpful only once.
But what's important and bugs me, is that debug log level somehow hinders `sccache` to communicate with redis.
Also, had to remove `sccache -s` as it was failing the job because it couldn't get the info from Redis. I tried some ideas, but think that in order to do that, we should install sccache to CI1 and CI2 which is not really necessary, at least for now.